pkg/segment/structs/segstructs_test.go

```diff
@@ -126,3 +126,69 @@ func Test_HasQueryAggergatorBlock(t *testing.T) {
 
 	assert.Equal(t, true, qa.HasQueryAggergatorBlock(), "Expected true when HasQueryAggergatorBlock is true, got false")
 }
+
+func Test_HasDedupBlock_NilQueryAggregators(t *testing.T) {
+	var qa *QueryAggregators
+
+	assert.Equal(t, false, qa.HasDedupBlock(), "Expected false when QueryAggregators is nil, got true")
+}
+
+func Test_HasDedupBlock_NilOutputTransforms(t *testing.T) {
+	var ot *OutputTransforms
+
+	qa := &QueryAggregators{
+		OutputTransforms: ot,
+	}
+
+	assert.Equal(t, false, qa.HasDedupBlock(), "Expencted false when OutputTransforms is nil, got true")
```

Comment:
typo: You have `Expencted` instead of `Expected`.

Also, you can remove these error messages in `assert.Equal()`; if an assert fails and there isn't a specified message, `assert` will print a simple message that's something like `expected: false, actual value: true`. So feel free to either keep or remove your custom messages, since they'll both provide the same information

Reply:
Thanks @AndrewHess for your comment, What if i will delete it, So should i delete an error message from HasQueryAggergatorBlock? cause i have put the error message too.

Reply:
Yep you can delete your error message instead of fixing the typo. When a custom error message isn't given, the `assert` package will generate one. Also the assert package (https://pkg.go.dev/github.com/stretchr/testify/assert) has some helpful methods like `True()` and `False()`, so here you could simply have `assert.False(t, qa.HasDedupBlock())` (and similar things for your other asserts)